### ISSUE TITLE
on Docker: ImportError: No module named flask_cors

### ISSUE THREAD

**Issue by strakaps** [hoangen/codeathon-backend#1]
Hi Hoang, I've run your Docker image, pulled the latest changes and it seems like the docker image does not contain the module flask_cors?

**Comment by hoangen** [comment]
Yes, that image still miss flask_cors library, you should run “pip install flask_cors” to install the missing library
